pkgs/applications/video/tartube/default.nix

```diff
@@ -32,7 +32,7 @@ python3Packages.buildPythonApplication rec {
   strictDeps = false;
 
   propagatedBuildInputs = with python3Packages; [
-    moviepy
+    (moviepy.override { inherit youtube-dl; })
```

Comment:
this will not work if https://github.com/NixOS/nixpkgs/pull/328432 gets merged

I think it would be better to fix moviepy itself

Reply:
#328418 doesn't really work at this moment.
BTW I think the best solution is to completely replace toutube-dl with yt-dlp in nixpkgs.